### PR TITLE
Introduce %y formatter

### PR DESCRIPTION
**Pull request by gwbres**
* Support %yy year on two digits

### CODE REVIEW

**Comment by ChristopherRabotin** [comment]
Looks good, thanks!

Closes #267

File: src/efmt/format.rs [hunk 1]
```diff
@@ -228,6 +230,12 @@ impl Format {
                 let sub_str = &s[prev_idx..end_idx];
 
                 match prev_token {
+                    Token::YearShort => {
+                        decomposed[0] = sub_str
+                            .parse::<i32>()
+                            .map_err(|_| Errors::ParseError(ParsingErrors::ValueError))?
+                            + 2000;
```
Comment (gwbres) [inline src/efmt/format.rs:237]: @ChristopherRabotin shouldn't `sub_str` be trimmed() here, so `.parse()` never fails ? 
In other terms, may whitespaces exist prior next separator

Comment (ChristopherRabotin) [reply src/efmt/format.rs:237]: Ah, I didn't think of this. I forget how it works. Don't we parse token by token and therefore it isn't possible to have a space? Do the other cases have a sub str? I'm afraid I'm busy this weekend and can't look into this.

**Comment by gwbres** [comment]
Hello, 
would it be possible to release this ?

**Comment by ChristopherRabotin** [comment]
Sure! I'll release it today.

**Comment by gwbres** [comment]
Hello Chris,
this won't buy me anything if the other issue is not solved, because my use case is to parse more easily GNSS timestamps and use the Day of Year aswell. So in my case, it can wait for its resolution

**Pull request merged**